examples/diff.c

```diff
@@ -290,9 +305,22 @@ static void parse_opts(struct opts *o, int argc, char *argv[])
 			o->findopts.flags |= GIT_DIFF_FIND_COPIES;
 		else if (!strcmp(a, "--find-copies-harder"))
 			o->findopts.flags |= GIT_DIFF_FIND_COPIES_FROM_UNMODIFIED;
-		else if (is_prefixed(a, "-B") || is_prefixed(a, "--break-rewrites"))
-			/* TODO: parse thresholds */
+		else if (optional_str_arg(
+				break_rewrite_arg, &args, "-B", NULL) || 
+			optional_str_arg(
+				break_rewrite_arg, &args, "--break-rewrites", NULL)) {
+
 			o->findopts.flags |= GIT_DIFF_FIND_REWRITES;
+			err = extract_break_rewrites(&m, &n, *break_rewrite_arg);
+			if (err==-1) 
```

Comment:
Also checking for the existence of `/` is not necessary as the `sscanf` functions returns how many parameters it has filled successfully. So if `/` is not there I will end up getting 0 and it will be handled accordingly.
If there's any other reason or correct testcase which might end up reported as error then there's a need to look into this.
You can do testing [here](http://ideone.com/1x0un0) if required.